static/riotjs/wizard/create_team.tag

```diff
@@ -129,14 +129,13 @@
                             if (student.team.id === self.team.id) {
                                 student.selected = true
                             }
-                            if (self.team.leader !== null) {
+                            if (!!self.team.leader) {
```

Comment:
Might be a good case for lodash `.get()` (especially being able to string together field names)

Reply:
You mean something like `if (_.get(self, 'team.leader'))` ?